examples/plot_expectation_maximisation_manifolds.py

```diff
@@ -0,0 +1,159 @@
+"""Apply Expectation Maximization on manifolds and plots the results.
+
+Random data is generated in seperate regions of the
+manifold. Then Expectation Maximization deduces a Gaussian Mixture Model
+that best fits the random data. For the moment
+the example works on the Poincaré Ball hyperbolic space.
+"""
+
+import os
+
+import matplotlib.pyplot as plt
+import mpl_toolkits.mplot3d.art3d as art3d
+from matplotlib.patches import Circle
```

Comment:
Ah nice! It makes me think that we should actually use this class `Circle` in the visualization module... instead  of reimplementing it! x)